zulipterminal/ui_tools/views.py

```diff
@@ -470,6 +482,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
 
 class UsersView(urwid.ListBox):
     def __init__(self, controller: Any, users_btn_list: List[Any]) -> None:
+        self.users_btn_list = users_btn_list
```

Comment:
This is part of the set_count refactor?

Reply:
Yes, this didn't have a button_list, should I move it to a separate commit?

Reply:
If this is only necessary for the set_count refactor, then it should go with those changes.